nixos/modules/services/networking/wg-quick.nix

```diff
@@ -265,9 +277,16 @@ let
 
         script = ''
           ${optionalString (!config.boot.isContainer) "modprobe wireguard"}
+          ${optionalString (values.configFile != null) ''
+            cp ${values.configFile} ${configPath}
+          ''}
           wg-quick up ${configPath}
         '';
 
+        serviceConfig = {
+          PrivateTmp = values.configFile != null;
```

Comment:
Don't we want hardening all the time!

Reply:
wg-quick just doesn't use /tmp in by itself, and since it's a short lived service — why would we?

Reply:
ghhh, okay, I still think it's a unnecessary nit, but whatever :shrug: